hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -625,6 +626,36 @@ private void initializeSystemManagers(OzoneConfiguration conf,
     }
 
     containerReplicaPendingOps = new ContainerReplicaPendingOps(conf, clock);
+
+    long intervalInMillis = conf.getTimeDuration(
+        ScmConfigKeys
+            .OZONE_SCM_EXPIRED_CONTAINER_REPLICA_OP_SCRUB_INTERVAL,
+        ScmConfigKeys
+            .OZONE_SCM_EXPIRED_CONTAINER_REPLICA_OP_SCRUB_INTERVAL_DEFAULT,
+        TimeUnit.MILLISECONDS);
+
+    long expiryMilliSeconds = conf.getTimeDuration(
```

Comment:
Just thinking that we may have different background threads if we reuse this class - maybe these variables should have a better name, eg containerReplicaOpExipryMs, containerReplicaOpScrubberIntervalMs.

waitTimeInMillis can be re-used for several services, `backgroundServiceSafemodeWaitMs`?